tests/test_capi.py

```diff
@@ -12,71 +12,6 @@
 ffi = cffi.FFI()
 
 
-def gen_classes():
```

Comment:
why did you remove this?

Reply:
It seems like it's not used anywhere in the tests

Reply:
It was a reminder to add tests ;-)  Could you try to add a basic initialization of each one of the classes?